### PR TITLE
Updating Gems List

### PR DESCRIPTION
**Pull request by kylekirkby**
Updating the list of required gems to reflect the recent updates made to ensure all sites are using the same version of the jumbo-jekyll-theme.

### CODE REVIEW

**Review by morancj — COMMENTED** [review]
LGTM

**Comment by kylekirkby** [comment]
@pcolmer @morancj - FYI that the test's don't appear to be running on bamboo for this PR...

**Comment by pcolmer** [comment]
It is building now, @kylekirkby. I had recently added "secrets" to the webhooks to make them more secure but I had forgotten about this one.

File: Dockerfile [hunk 1]
```diff
@@ -123,28 +123,13 @@ ENV RUBY_GEMS \
...
- jumbo-jekyll-theme:5.6.4 \
- # Used by 96boards.org
- jumbo-jekyll-theme:5.5.3 \
- # Used by devicetree.org, linaro.cloud, linaro.org,
- # mlplatform.org, op-tee.org, trustedfirmware.org,
- jumbo-jekyll-theme:5.5.1 \
- # (staging).openampproject.org
- jumbo-jekyll-theme:5.6.5 \
- # Used by staging.linaro.org
+ # Used by all Jekyll sites
```
Comment (pcolmer) [inline Dockerfile:126]: Except for LKFT :)

**Review by pcolmer — APPROVED** [review]

**Pull request merged**